idd/Energy+.idd.in

```diff
@@ -34502,6 +34502,4630 @@ ZoneHVAC:EvaporativeCoolerUnit,
+		\units minutes
+		\default 10
+	A13,	\field First fuel type
+		\type choice
+		\key Electricity
+		\default Electricity
+		\note Select the fuel type associated with field: "System Electric Power Lookup Table" in each mode. 
+	A14,	\field Second fuel type
+		\type choice
+		\key None
+		\key Electricity
+		\key NaturalGas
+		\key PropaneGas
+		\key FuelOil#1
+		\key FuelOil#2
+		\key Diesel
+		\key Gasoline
+		\key Coal
+		\key OtherFuel1
+		\key OtherFuel2
+		\key Steam
+		\key DistrictHeating
+		\key DistrictCooling
+		\note Select the fuel type associated with field: "System Second Fuel Consumption Lookup Table" in each mode.
+		\note If this field is blank, default second fuel type = None.
+		\default None
+	A15,	\field Third fuel type
+		\type choice
+		\key None
+		\key Electricity
+		\key NaturalGas
+		\key PropaneGas
+		\key FuelOil#1
+		\key FuelOil#2
+		\key Diesel
+		\key Gasoline
+		\key Coal
+		\key OtherFuel1
+		\key OtherFuel2
+		\key Steam
+		\key DistrictHeating
+		\key DistrictCooling
+		\note Select the fuel type associated with field: "System Third Fuel Consumption Lookup Table" in each mode.
+		\note If this field is blank, default third fuel type = None.
+		\default None
+	A16,	\field Objective Function Minimizes
+		\type choice
+		\key Electricity Use
+		\key Second Fuel Use
+		\key Third Fuel Use
+		\key Water Use
+		\note In each time step, controlled variables will be chosen to minimize the selection in this field, subject to constraints.
+		\note If this field is blank, the objective function will minimize electricity use.
+		\default Electricity Use
+	A17,	\field Design Specification Outdoor Air Object Name
+		\type object-list
+		\object-list DesignSpecificationOutdoorAirNames
+		\note Enter the name of a DesignSpecification:OutdoorAir object. Information in that object will be used to compute the minimum outside air flow rate in each time step.
+		\note If this field is blank, the system may still supply outside air, if it is capable as described by lookup tables, when doing so is the most efficient way to satisfy other constraints.
+	A18,	\field Mode0 Name
+		\type alpha
+		\retaincase
+		\note Enter a name for Mode0.
+		\note Mode0 describes equipment performance in standby.  Mode0 is usually characterized by electricity use for controls and crankcase heaters, or other standby resouce consumption. Mode0 will be chosen for any timestep, or portion of timestep, when no ventilation, cooling, or dehumidification is required.  Mode0 is available at any environmental condition.
+	A19,	\field Mode0 Supply Air Temperature Lookup Table Name
```

Comment:
Will this model restrict use of performance curves? Tables and curves are interchangeable in E+. Also, if tables or curves are used, what is the need for max/min temperature limit inputs above?